tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/observability/tracing/RemoteCallTestCase.java

```diff
@@ -181,20 +191,25 @@ public void testRemoteFunctionErrorReturn(String resourceName, String resourceFu
         span1.ifPresent(span -> {
             Assert.assertTrue(spans.stream().noneMatch(mockSpan -> mockSpan.getTraceId() == traceId
                     && mockSpan.getSpanId() == span.getParentId()));
-            Assert.assertEquals(span.getOperationName(), resourceName);
-            Assert.assertEquals(span.getTags(), toMap(
-                    new AbstractMap.SimpleEntry<>("span.kind", "server"),
-                    new AbstractMap.SimpleEntry<>("src.module", DEFAULT_MODULE_ID),
-                    new AbstractMap.SimpleEntry<>("src.position", resourceFunctionPosition),
-                    new AbstractMap.SimpleEntry<>("src.service.resource", "true"),
-                    new AbstractMap.SimpleEntry<>("http.url", "/" + SERVICE_NAME + "/" + resourceName),
-                    new AbstractMap.SimpleEntry<>("http.method", "POST"),
-                    new AbstractMap.SimpleEntry<>("protocol", "http"),
-                    new AbstractMap.SimpleEntry<>("service", SERVICE_NAME),
-                    new AbstractMap.SimpleEntry<>("resource", resourceName),
-                    new AbstractMap.SimpleEntry<>("src.object.name", SERVER_CONNECTOR_NAME),
-                    new AbstractMap.SimpleEntry<>("error", "true")
-            ));
+            Assert.assertEquals(span.getOperationName(), "post /" + resourceName);
+//            TODO: Uncomment this assertion once #ballerina-lang/issues/28686 is fixed
+//            Assert.assertEquals(span.getTags(), toMap(
+//                    new AbstractMap.SimpleEntry<>("span.kind", "server"),
+//                    new AbstractMap.SimpleEntry<>("src.module", DEFAULT_MODULE_ID),
+//                    new AbstractMap.SimpleEntry<>("src.position", resourceFunctionPosition),
+//                    new AbstractMap.SimpleEntry<>("src.service.resource", "true"),
+//                    new AbstractMap.SimpleEntry<>("http.url", "/" + SERVICE_NAME + "/" + resourceName),
+//                    new AbstractMap.SimpleEntry<>("http.method", "POST"),
+//                    new AbstractMap.SimpleEntry<>("protocol", "http"),
+//                    new AbstractMap.SimpleEntry<>("src.object.name", SERVICE_NAME),
+//                    new AbstractMap.SimpleEntry<>("listener.name", SERVER_CONNECTOR_NAME),
+//                    new AbstractMap.SimpleEntry<>("entrypoint.function.module", DEFAULT_MODULE_ID),
+//                    new AbstractMap.SimpleEntry<>("entrypoint.function.position", resourceFunctionPosition),
+//                    new AbstractMap.SimpleEntry<>("error", "true"),
+//                    new AbstractMap.SimpleEntry<>("src.resource.accessor", "post"),
+//                    new AbstractMap.SimpleEntry<>("error.message", errorMessage),
+//                    new AbstractMap.SimpleEntry<>("src.resource.path", "/" + resourceName)
+//            ));
```

Comment:
Shall we keep everything else and only comment the assertion for `error.message` tag ?

Reply:
Please fix in all other relevant places.

Reply:
FIxed with 3a5fad4b8ee95545ae441daea7a28e24052367f6